### PR TITLE
18 Implement `Presentation.Share` SDK Function

### PR DESCRIPTION
**Pull request by LuisOsta**

### CODE REVIEW

File: service/credential_adapter/credential_adapter.go [hunk 1]
```diff
@@ -135,8 +136,12 @@ func (c *credentialAdapterClient) PresentVerifiableCredential(creds []model.Seri
 
 	for i, cred := range creds {
 
+		encoded_cred, err := json.Marshal(cred)
```
Comment (sKudryashov) [inline service/credential_adapter/credential_adapter.go:139]: Var name `encodedCred`

**Review by sKudryashov — APPROVED** [review]
LGTM but varname can be changed

File: service/credential_adapter/credential_adapter.go (hunk 1 above)
Comment (LuisOsta) [reply service/credential_adapter/credential_adapter.go:139]: Sounds good

Comment (LuisOsta) [reply service/credential_adapter/credential_adapter.go]: Fixed

**Pull request merged**